docs/source/concept_models/vivarium_iv_iron/child_model/output_table.csv

```diff
@@ -1,12 +1,12 @@
-Measure,Child age/sex/year stratification
-Child stunting state person time,TRUE
-Child wasting state person time,TRUE
-Cause model state transition count,TRUE
-Cause model state person time,TRUE
-Cause-specific YLDs,TRUE
-Cause-specific YLLs,TRUE
-Cause-specific deaths,TRUE
-Sum of birthweight at birth,TRUE
-Sum of gestational age at birth,TRUE
-Count of live births,TRUE
-Count of live births with birthweight less than 2500 g,TRUE
+Measure,Child age/sex/year stratification,IFA coverage in baseline scenario
```

Comment:
I don't understand what this new column means

Reply:
The measures with "TRUE" for that column should be stratified by IFA coverage for the baseline scenario. Does that make sense now?

![image](https://user-images.githubusercontent.com/54918472/165349861-6f43750d-1999-4dc2-954c-f082f2d1a208.png)